### DISCUSSION TITLE
Figma File

### DISCUSSION THREAD

**Discussion by johnfabian** [primefaces/community#1767] [PrimeVue Templates] [answered]
I purchased the Ultima template and says on the main page Ultima uses Figma as the design tool. It will be possible to download the Figma file after your purchase. You can [preview the Figma file](https://www.figma.com/file/ijQrxq13lxacgkb6XHlLxA/Preview-%7C-Ultima-2022?node-id=354%3A7715&t=gjWHprUDE5RJIg78-1) before the purchase

I'm not able to download the figma file, where can I get this at? I can't keep waiting for support, please help or give a contact email or support

**Comment by cagataycivici** [comment, marked as answer]
<img width="369" alt="Screenshot 2024-04-30 at 14 41 18" src="https://github.com/primefaces/community/assets/686247/1a60be2c-032c-44b4-a624-855e06556b82">

Please check the Figma tab at the download dialog.

**Reply by johnfabian** [reply]
I see it now thank you
